content/id/docs/concepts/workloads/controllers/jobs-run-to-completion.md

```diff
@@ -0,0 +1,506 @@
+dan memastikan nilainya tidak akan beririsan dengan Job lainnya.
+
+Meskipun demikian, ipada beberapa kasus, kamu bisa saja memiliki kebutuhan untuk meng-_override_ 
+nilai dari selektor ini. Untuk melakukannya, kamu dapat menspesifikasikan `.spec.selector`
+dari Job.
+
+Berhati-hatilah ketika kamu melakukan proses ini. Jika kamu menspesifikasikan sebuah label 
+selektor yang tidak unik pada Pod yang ada di dalam Job tersebut, serta sesuai dengan Pod yang tidak 
+terkait dengan Job tadi, maka Pod dari Job yang tidak terkait dengan Job tadi akna dihapus, atau Job ini 
+akan menghitung _completion_ dari Pod lain sebagai tolak ukur suksesnya Job tersebut, atau bisa saja salah satu 
+atau kedua Job tidak dapat membuat Pod baru yang digunakan untuk menyelesaikan Job tersebut. 
+Jika selektor yang tidak unik dipilih, maka kontroler lain (misalnya ReplicationController) dan Pod 
+yang ada di dalamnya bisa saja memiliki perilaku yang tidak dapat diprediksi. Kubernetes tidak akan 
+mencegah kemungkinan terjadinya hal ini ketika kamu menspesifikasikan nilai `.spec.selector`.
+
+Berikut merupakan contoh skenario dimana kamu ingin menggunakan fitur ini.
+
+Misalnya saja Job dengan nama `old` sudah dijalankan. 
+Dan kamu ingin Pod yang sudah dijalankan untuk tetap berada pada state tersebut, 
+tapi kamu juga ingin Pod selanjutnya yang dibuat untuk menggunakan templat Pod yang berbeda dan agar 
+Job tersebut memiliki nama yang berbeda. Kamu tidak dapat mengubah Job karena _field_ ini 
+merupakan nilai yang tidak bisa diubah. Dengan demikian, kamu menghapus Job `old` 
+tetapi tetap membiarkan Pod yang ada untuk jalan, menggunakan perintah `kubectl delete jobs/old --cascade=false`.
+Sebelum menghapus Job tadi, kamu mencatat selektor yang digunakan oleh Job tadi:
+
+```
+kubectl get job old -o yaml
+```
+```
+kind: Job
+metadata:
+  name: old
+  ...
+spec:
+  selector:
+    matchLabels:
+      controller-uid: a8f3d00d-c6d2-11e5-9f87-42010af00002
+  ...
+```
+
+Kemudian kamu membuat sebuah Job baru dengan nama  `new` 
+dan kamu secara eksplisit menspesifikasikan selektor yang sama.
+Karena Pod dengan selektor yang sama memiliki label `controller-uid=a8f3d00d-c6d2-11e5-9f87-42010af00002`,
+maka Pod-Pod lama tadi dikendalikan juga oleh Job `new`.
+
+Kamu harus menspesifikasikan `manualSelector: true` pada Job yang baru 
+karena kamu tidak menggunakan selektor yang diberikan secara default oleh sistem.
+
+```
+kind: Job
+metadata:
+  name: new
+  ...
+spec:
+  manualSelector: true
+  selector:
+    matchLabels:
+      controller-uid: a8f3d00d-c6d2-11e5-9f87-42010af00002
+  ...
+```
+
+Job yang baru tadi kemudian akan memiliki uid yang berbeda dari `a8f3d00d-c6d2-11e5-9f87-42010af00002`.  Pengaturan 
+`manualSelector: true` memberikan perintah pada sistem bahwa kamu mengetahui apa yang kamu lakukan 
+dan untuk mengizikan ketidaksesuaian ini untuk terjadi.
+
+## Alternatif
+
+### _Bare Pods_
```

Comment:
How about translating bare to Polos or something else, this bare pod means running pod without any deployment, daemonset, etc.
```suggestion
### _Pod Polosan_
```